sale_comment_template/views/base_comment_template_view.xml

```diff
@@ -5,11 +5,8 @@
         <field name="type">ir.actions.act_window</field>
         <field name="res_model">base.comment.template</field>
         <field name="view_mode">tree,form</field>
-        <field name="domain">[('model_ids.model', '=', 'sale.order')]</field>
-        <field
-            name="context"
-            eval="{'default_model_ids': [(4, ref('sale.model_sale_order'))]}"
-        />
+        <field name="domain">[('models', '=', 'sale.order')]</field>
+        <field name="context" eval="{'default_models': 'sale.order'}" />
```

Comment:
```suggestion
        <field name="context">{'default_models': 'sale.order'}</field>
```
`eval` was needed for `eval`uating `ref`; it works either way so this is not blocking.